app/src/main/java/com/sethpthomas91/httpserver/response/Body.java

```diff
@@ -8,6 +8,14 @@ public Body(String uniformResourceIdentifier) {
             this.bodyText = "Hello world";
         } else if (uniformResourceIdentifier.equals("/echo_body")) {
             this.bodyText = "some body";
+        } else if (uniformResourceIdentifier.equals("/text_response")) {
+            this.bodyText = "text response";
+        } else if (uniformResourceIdentifier.equals("/html_response")) {
+            this.bodyText = "<html><body><p>HTML Response</p></body></html>";
+        } else if (uniformResourceIdentifier.equals("/json_response")) {
+            this.bodyText = "{\"key1\":\"value1\",\"key2\":\"value2\"}";
+        } else if (uniformResourceIdentifier.equals("/xml_response")) {
+            this.bodyText = "<note><body>XML Response</body></note>";
```

Comment:
This conditional here still feels a little awkward, but glad to see a pattern developing.